packages/blade/src/components/Alert/styles.ts

```diff
@@ -26,6 +26,7 @@ export const getCommonStyles = (props: StyledProps<StyledAlertProps>): CSSObject
     display: 'flex',
     flexDirection: 'row',
     maxWidth: isFullWidth ? 'auto' : makeSize(MAX_WIDTH),
+    width: isFullWidth ? '100%' : undefined,
```

Comment:
🚨 Important: Added width because Alert would require explicit width when `position="absolute"` is passed

Reply:
Moved to #1016